src/components/Navbar.js

```diff
@@ -0,0 +1,56 @@
+import React, { useState } from 'react';
+import { NavLink } from 'react-router-dom';
+import { MdClose } from 'react-icons/md';
+import { FiMenu } from 'react-icons/fi';
+// eslint-disable-next-line
```

Comment:
Kindly consider fixing the errors instead of disabling linters as they help us to follow good coding practices and writing clean codes.